core/src/classes/codeConfig.ts

```diff
@@ -360,20 +360,26 @@ export async function getParentIds(
     providedIds.push(`property:${configObject["bootstrappedProperty"].id}`);
   }
 
-  // - query property with mustache dependency
-  if (
-    cleanClass(configObject) === "property" &&
-    configObject["options"] &&
-    configObject["options"]["query"]
-  ) {
-    const mustachePrerequisiteIds =
-      await MustacheUtils.getMustacheVariablesAsPropertyIds(
-        configObject["options"]["query"],
-        otherConfigObjects
-      );
-    prerequisiteIds.push(
-      ...mustachePrerequisiteIds.map((p) => `property:${p}`)
-    );
+  // - property with mustache dependency
```

Comment:
now will sort ANY property with a mustache dependency, not just query source properties